src/homework6/decorator.py

```diff
@@ -0,0 +1,47 @@
+"""Создайте декоратор, который вызывает задекорированную функцию
+пока она не выполнится без исключений (но не более n раз - параметр
+декоратора). Если превышено количество попыток, должно быть возбуждено
+исключение типа TooManyErrors
+
+"""
+
+
+def decorator(errors):
+
+    def dec(func):
+        def wrapper(*args, **kwargs):
+            nonlocal errors
```

Comment:
why it's nonlocal? we don't have it in enclosure scope

Reply:
cleared